circuit/src/ecc/native.rs

```diff
@@ -50,6 +56,41 @@ where
 		Self::new(Integer::one(), Integer::one())
 	}
 
+	/// Selection function for the table
+	fn select(bit: bool, table: [Self; 2]) -> Self {
+		if bit {
+			table[1].clone()
+		} else {
+			table[0].clone()
+		}
+	}
+
+	/// AuxInit
+	pub fn to_add() -> Self {
+		let x = BigUint::from_str(
+			"16674715582331070136933915527288889019778094566732803816939061043898457168778",
+		)
+		.unwrap();
+		let y = BigUint::from_str(
+			"6463266180564494723216328990756243248194282857153665680958564348491911026035",
+		)
+		.unwrap();
+		Self::new(Integer::new(x), Integer::new(y))
+	}
+
+	/// AuxFin
+	pub fn to_sub() -> Self {
+		let x = BigUint::from_str(
+			"13309346756614635333825509300060708409153440696756098989763444826124962874203",
+		)
+		.unwrap();
+		let y = BigUint::from_str(
+			"19567858517750281949071914889393977810091135383309479881360435638939337926797",
+		)
+		.unwrap();
+		Self::new(Integer::new(x), Integer::new(y))
+	}
```

Comment:
To avoid parsing these numbers and decomposing them every time, we should have hardcoded limbs here.
Also we should consider moving this to Rns.